webapp/src/app.tsx

```diff
@@ -1531,6 +1532,24 @@ ${compileService ? `<p>${lf("{0} version:", "C++ runtime")} <a href="${Util.html
         pxt.tickEvent(`tutorial.showhint`, { tutorial: options.tutorial, step: options.tutorialStep });
     }
 
+    showHiddenMenu = (e: React.KeyboardEvent) => {
+        let charCode = (typeof e.which == "number") ? e.which : e.keyCode
```

Comment:
This isn't the right way to do this. The best way to do this is with CSS. I see why the menu only shows up once you've already focused on the logo. 

What you want to do is do what Mozilla does on mdn, and add the menu in the DOM, if it's the first one it'll be the first to get focus. and then you hide the menu, and only show it on focus with CSS.

```
#accessibilityMenu {
    display:none;
}
#accessibilityMenu:focus {
   display: block;
}
```